### ISSUE TITLE
Build error

### ISSUE THREAD

**Issue by bmdelacruz** [PureSwift/Cacao#27]
I'm trying to run your demo application but the build failed because of the error below.

```
Compile Swift Module 'Silica' (19 sources)
/home/bmdelacruz/Documents/Projects/swift/Cacao/.build/checkouts/Silica.git-4667709516086632535/Sources/Silica/CGRect.swift:32:56: error: ambiguous use of 'nan'
    public static var null: CGRect { return CGRect(x: .nan, y: .nan, width: .nan, height: .nan) }
                                                       ^
Foundation.CGFloat:14:23: note: found this candidate
    public static var nan: Foundation.CGFloat { get }
                      ^
Swift.Double:137:23: note: found this candidate
    public static var nan: Double { get }
```

OS: Ubuntu 16.04
Swift version: Swift version 4.0-dev (LLVM 2dedb62a0b, Clang b9d76a314c, Swift 61ad3c07ef)

**Comment by lilianmoraru** [comment]
Same here, with the same version of `Swift` :)

**Comment by ardawan** [comment]
I got this error too. on Debian 9, swift 4

**Comment by Joe-Soap** [comment]
I had the very same problem with uBuntu 16.10 and swift 4. I had to go through the entire list of errors and correct them. Once you sort out the above errors, you are faced with many “Ambiguous” errors that are a chore to correct. Is PureSwift an active repository. 

Once I did get it to run. I don’t know if it is my laptop (old amd64 HP) or the code, but it runs at a snails pace.

**Comment by colemancda** [comment]
Please only download the stable releases, I'm gonna try to move to GitFlow and work on `develop` after the next release. Thanks!

**Comment by colemancda** [comment]
The `master` branch is now buildable on macOS as of 55ecb28c561bd7d56d0a92aa5dd2cc753b80e628, and I created a new `develop` branch for WIP.

**Comment by colemancda** [comment]
[v0.7.0](https://github.com/PureSwift/Cacao/releases/tag/0.7.0) Fixed these builds errors for both the Swift 3.0.2 & 4.0.2 compilers on Linux.
